### PR TITLE
task 1 completed

### PR DESCRIPTION
**Pull request by prativa222**

### CODE REVIEW

**Review by Anubhcv — APPROVED** [review]

**Review by Anubhcv — CHANGES_REQUESTED** [review]
Review your task. You have pushed extra files.

**Comment by prativa222** [comment]
> Review your task. You have pushed extra files.

bhaiya jo extra files pull hue h use undo kese karu

**Comment by Anubhcv** [comment]
> > Review your task. You have pushed extra files.
> 
> bhaiya jo extra files pull hue h use undo kese karu

Delete the extra files from your local repo then commit and push.

File: Open Source/Git1/prativarath.txt
Comment (ashuvssut) [inline Open Source/Git1/prativarath.txt:3]: .

**Comment by prativa222** [comment]
> > > Review your task. You have pushed extra files.
> > 
> > 
> > bhaiya jo extra files pull hue h use undo kese karu
> 
> Delete the extra files from your local repo then commit and push.

bhaiya pata nhi chal rhi h konsi extra file push hue h or me task 1 submit karchuki hu par meri name show nhi kr rhi h name list me

**Comment by Anubhcv** [comment]
In git2 folder you have uploaded an extra file. 
Your name will show in name list when we merge this pr after your task 2 gets approved.

**Review by Anubhcv — APPROVED** [review]

**Comment by prativa222** [comment]
> In git2 folder you have uploaded an extra file.
> Your name will show in name list when we merge this pr after your task 2 gets approved.

Bhaiya task 2 submit ho gayi na meri ? Pata nhi chal rhi h..

**Pull request merged**
